api/resources/translations/messages-ne.properties

```diff
@@ -44,24 +51,24 @@ Confirm = पुष्टी गर्नुहोस्
 Confirm\ Password = पासवर्ड पृष्टि गर्नु
```

Comment:
Shouldn't this be पुष्टी instead of पृष्टि ?

Reply:
I refer to this dictionary, which is very similar to the official one Nepali Brihat Sambdakosh, it says [`पुष्टि`](https://nepalishabdakosh.com/dictionary/NepNep/%E0%A4%AA%E0%A5%81%E0%A4%B7%E0%A5%8D%E0%A4%9F%E0%A5%80/%E0%A4%AA%E0%A5%81%E0%A4%B7%E0%A5%8D%E0%A4%9F%E0%A5%80-meaning-in-nepali).

Reply:
@binokaryg I meant changing पृष्टि